docs/ceph-csi-upgrade.md

```diff
@@ -345,3 +352,15 @@ reboot.
 It is suggested to upgrade to the latest ceph-csi release and drain application
 pods on all the nodes so that new mount option `_netdev` can be added to all
 the mountpoints.
+
+### CSI Sidecar containers consideration
+
+With 3.2.0 version of ceph-csi we have updated the versions of CSI sidecar
+containers. These versions are generally compatible with kubernetes
+version>=1.17 but based on the kubernetes version you are using, you need to
+update the templates with required sidecar versions.
+You also might need to update or remove a few arguments based on the sidecar
+versions you are using.
+Refer
```

Comment:
Refer [sidecar-compatibility](https://kubernetes-csi.github.io/docs/sidecar-containers.html) for mode details?

Reply:
fixed